sdk/trace/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpanProcessor.java

```diff
@@ -165,17 +170,22 @@ public String toString() {
     private volatile boolean continueWork = true;
     private final ArrayList<SpanData> batch;
 
+    private final Set<CompletableResultCode> activeExports =
+        Collections.newSetFromMap(new ConcurrentHashMap<>());
```

Comment:
NIT: ConcurrentHashMap.newKeySet()

Reply:
animal sniffer said no, must not be available in android 21